### PR TITLE
feat(crux-ui): add form submit to ctrl+enter

### PR DESCRIPTION
**Pull request by bvektor**
- dyo input id
- dyo text area id

### CODE REVIEW

File: web/crux-ui/src/elements/dyo-form.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+export interface DyoFormProps extends Omit<React.TextareaHTMLAttributes<HTMLFormElement>, 'ref'> {}
+
+const DyoForm = (props: DyoFormProps) => {
+  const { onSubmit, onKeyDown, children, ...forwardedProps } = props
+
+  const keydownHandler = e => {
+    if (e.key === 'Enter' && e.ctrlKey) onSubmit(e)
```
Comment (polaroi8d) [inline web/crux-ui/src/elements/dyo-form.tsx:9]: Will it work with the macOS command key?

Comment (polaroi8d) [reply web/crux-ui/src/elements/dyo-form.tsx:9]: Talked in person - @bvektor will complete with the command key.

**Review by polaroi8d — APPROVED** [review]
Minor change, otherwise LGTM

File: web/crux-ui/src/components/shared/secret-key-value-input.tsx [hunk 2]
```diff
@@ -298,9 +298,7 @@ const SecretKeyValInput = (props: SecretKeyValueInputProps) => {
   return (
     <form className={clsx(className, 'flex flex-col max-h-128 overflow-y-auto')}>
       {!label ? null : (
-        <DyoLabel className={clsx(labelClassName ?? 'text-bright mb-2 whitespace-nowrap text-light-eased')}>
-          {label}
-        </DyoLabel>
+        <DyoLabel className={clsx(labelClassName ?? 'mb-2 whitespace-nowrap text-light-eased')}>{label}</DyoLabel>
```
Comment (m8vago) [inline web/crux-ui/src/components/shared/secret-key-value-input.tsx:301]: label is always exist, i suppose?

File: web/crux-ui/src/elements/dyo-file-upload.tsx [hunk 3]
```diff
@@ -43,6 +43,7 @@ export const DyoFileUploadInput = (props: DyoFileUploadProps) => {
       <input
         type="file"
         className="hidden"
+        id={name}
```
Comment (m8vago) [inline web/crux-ui/src/elements/dyo-file-upload.tsx:46]: why is this necessary?

File: web/crux-ui/src/elements/dyo-input.tsx [hunk 4]
```diff
@@ -46,6 +46,7 @@ export const DyoInput = forwardRef((props: DyoInputProps, ref: ForwardedRef<HTML
           name={name}
           ref={ref}
           disabled={disabled}
+          id={name}
```
Comment (m8vago) [inline web/crux-ui/src/elements/dyo-input.tsx:49]: I've just noticed you are using this for tests.
That's something we're trying to avoid.
Please resolve those elements in the test some other way.

**Review by peterambrus3 — APPROVED** [review]
tested, works fine
